pkg/issuer/venafi/client/fake/BUILD.bazel

```diff
@@ -6,10 +6,10 @@ go_library(
         "connector.go",
         "venafi.go",
     ],
-    importpath = "github.com/jetstack/cert-manager/pkg/internal/venafi/fake",
+    importpath = "github.com/jetstack/cert-manager/pkg/issuer/venafi/client/fake",
     visibility = ["//pkg:__subpackages__"],
```

Comment:
This needs updating to be public visibility since the package has moved out of `internal/`

Reply:
Doesn't seem to be resolved

Reply:
oops fixed everywhere but the fake, fixed now